README.md

```diff
@@ -0,0 +1,19 @@
+[![CircleCI](https://circleci.com/gh/bids-standard/pybv.svg?style=svg)](https://circleci.com/gh/bids-standard/pybv)
+
+# pyBV
+
+A lightweight I/O utility for the BrainVision data format, written in Python.
+
+**See the [pybv documentation](https://pybv.readthedocs.io) for more
+information.**
+
+**ALPHA SOFTWARE**. This package is currently in its early stages of iteration.
+It may change both its internals or its user-facing API in the near future. Any
+feedback and ideas on how to improve either of these is more than welcome!
+
+## Acknowledgements
+
+This package was originally adapted from [palday](https://github.com/palday)'s
```

Comment:
https://gitlab.com/palday

Reply:
```suggestion
This package was originally adapted from [palday](https://gitlab.com/palday)'s
```

Reply:
I think it's probably "https://palday.bitbucket.io/" if we want to be very strict :-D